terraform/cloud.gov.tf

```diff
@@ -58,7 +58,7 @@ resource "aws_route53_record" "cloud_gov_cloud_gov_txt" {
   name = "cloud.gov."
   type = "TXT"
   ttl = 300
-  records = ["v=spf1 include:spf.mandrillapp.com ?all"]
+  records = ["v=spf1 include:spf.mandrillapp.com -all"]
```

Comment:
if we're expecting failure here, do we have anything tracking this? or will it be a silent failure?

Reply:
We should be able to track deliverability through senders we know we use (mailchimp/mandrill), but tracking/reporting for unknown senders isn't really available without DMARC.